cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 8/13/2024_
 
+**Performance:**
+
+- Fixed a potential memory leak when re-connecting to an unintentionally disconnected CDP connection. Fixes [#29744](https://github.com/cypress-io/cypress/issues/29744). Addressed in [#29988](https://github.com/cypress-io/cypress/pull/29988)
```

Comment:
Do we want to call out that this is a server side leak? Maybe to help avoid confusion with some of the browser-side ones we've got going on.